$NetBSD: patch-ac,v 1.1 2007/11/07 12:28:59 rillig Exp $

xdmain.c:1390: error: static declaration of 'xd_handle_table' follows non-static declaration

--- xdmain.c.orig	1999-04-10 02:56:00.000000000 +0200
+++ xdmain.c	2007-11-07 13:26:28.000000000 +0100
@@ -49,7 +49,7 @@
 
 #include "xdelta.h"
 
-extern HandleFuncTable xd_handle_table;
+static HandleFuncTable xd_handle_table;
 
 #define XD_PAGE_SIZE (1<<20)
 
